README.md

```diff
@@ -8,135 +8,62 @@ Watch an introduction to TigerBeetle on [Zig SHOWTIME](https://www.youtube.com/w
+    ledger: 1,   // u32, ledger value
+    code: 718, // u16, a chart of accounts code describing the type of account (e.g. clearing, settlement)
+    flags: 0,  // u16
+    debits_pending: 0n,  // u64
+    debits_posted: 0n,  // u64
+    credits_pending: 0n, // u64
+    credits_posted: 0n, // u64
+    timestamp: 0n, // u64, Reserved: This will be set by the server.
+  },
+]);
 ```
 
-*The VOPR* stands for *The Viewstamped Operation Replicator* and was inspired by the movie WarGames, by our love of fuzzing over the years, by [Dropbox's Nucleus testing](https://dropbox.tech/infrastructure/-testing-our-new-sync-engine), and by [FoundationDB's deterministic simulation testing](https://www.youtube.com/watch?v=OJb8A6h9jQQ).
-
-*The VOPR* is [a deterministic simulator](src/simulator.zig) that can fuzz many clusters of TigerBeetle servers and clients interacting through TigerBeetle's Viewstamped Replication consensus protocol, but all within a single developer machine process, with [a network simulator](src/test/packet_simulator.zig) to simulate all kinds of network faults, and with an in-memory [storage simulator](src/test/storage.zig) to simulate all kinds of storage faults, to explore and test TigerBeetle against huge state spaces in a short amount of time, by literally speeding up the passing of time within the simulation itself.
-
-Beyond being a deterministic simulator, *The VOPR* also features [a state checker](src/test/state_checker.zig) that can hook into all the replicas, and check all their state transitions the instant they take place, using cryptographic hash chaining to prove causality and check that all interim state transitions are valid, based on any of the set of inflight client requests at the time, without divergent states, and then check for convergence to the highest state at the end of the simulation, to distinguish between correctness or liveness bugs.
-
-Check out TigerBeetle's [Viewstamped Replication Made Famous](https://github.com/coilhq/viewstamped-replication-made-famous#how-can-i-run-the-implementation-how-many-batteries-are-included-do-you-mean-i-can-even-run-the-vopr) bug bounty challenge repository for more details on how to run *The VOPR* and interpret its output.
+For further reading:
 
-## Launch a Local Cluster
-
-Launch a TigerBeetle cluster on your local machine by running each of these commands in a new terminal tab:
-
-```
-./tigerbeetle format --cluster=0 --replica=0 0_0.tigerbeetle
-./tigerbeetle format --cluster=0 --replica=1 0_1.tigerbeetle
-./tigerbeetle format --cluster=0 --replica=2 0_2.tigerbeetle
-
-./tigerbeetle start --addresses=3001,3002,3003 0_0.tigerbeetle
-./tigerbeetle start --addresses=3001,3002,3003 0_1.tigerbeetle
-./tigerbeetle start --addresses=3001,3002,3003 0_2.tigerbeetle
-```
-
-Run the TigerBeetle binary to see all command line arguments:
-
-```bash
-./tigerbeetle --help
-```
+* [Running a 3-node cluster locally with docker-compose](./docs/DOCKER_COMPOSE.md)
 
 ## Clients
```

Comment:
There's also https://github.com/coilhq/tigerbeetle-go now.
Also, I noticed (below this) there's a line that says

> [client.zig](./src/vr/client.zig) is a TigerBeetle Zig client.


...which is true but maybe a little misleading. `client.zig` is `StateMachine`-agnostic — it is lower level than `tigerbeetle-node`.

In other words, `client.zig` doesn't know anything about Accounts or Transfers, only Messages. It is a library for building a client implementation, but not something to use on its own.

(Also, the path is `/vsr`, not `/vr`).

Reply:
Dropped the Zig client and added the Go client. Thank you!